### DISCUSSION TITLE
std::format - how to best use with a runtime fmt string?

### DISCUSSION THREAD

**Discussion by nuuSolutions** [microsoft/STL#4099] [Q&A] [answered]
seems undocumented

**Comment by sylveon** [comment]
std::vformat

**Comment by nuuSolutions** [comment]
for `vformat` one has to manually `std::make_format_args` which makes it rather seem like "not an interface"

**Reply by sylveon** [reply, marked as answer]
Because the format string is provided at runtime, it makes sense for the format args to also be provided at runtime and not statically determined, that's what std::make_format_args is for.
